reyna/src/com/b2msolutions/reyna/services/StoreService.java

```diff
@@ -38,28 +37,59 @@ public static void setCellularDataBlackout(Context context, TimeRange range) {
         new Preferences(context).saveCellularDataBlackout(range);
     }
 
-	@Override
-	protected void onHandleIntent(Intent intent) {
+    public static void setStorageSizeLimit(Context context, long limit) {
+        if (limit <= 0) {
```

Comment:
Why are we actually ignoring limit <= 0, rather than setting it to minimum_storage_limit as in line 45? I assume that "limit" here is whatever the user has set in the configuration minus the space we reserve for Elemez itself, right? That means limit could become negative even if the user has set the config to something positive. As the amount reserved by Elemez is not really transparent to the user, probably just setting it to the minimum is the best in this case.